### PR TITLE
feat(EMS-3228): No PDF - Check your answers - Export contract - Summary list

### PR DESCRIPTION
**Pull request by ttbarnes**
## Introduction :pencil2:
This PR sets up the summary list for the "Check your answers - Export contract" page of a No PDF application.

## Resolution :heavy_check_mark:
- Add E2E tests.
- Create new DRY E2E assertions for the "Export contract" summary list:
  - `assertGenericExportContractSummaryListRows`
  - `assertMinimalExportContractSummaryListRows`
  - `assertFullyPopulatedExportContractSummaryListRows`
- Update existing Export contract summary list tests to use the above DRY assertions.
- Update the UI GET controller to generate and render a summary list.
- Update `test.yml`

## Miscellaneous :heavy_plus_sign:
- Fix some typos.

### CODE REVIEW

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-minimal.spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  CHECK_YOUR_ANSWERS: { EXPORT_CONTRACT },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswers;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Check your answers - Export contract - As an exporter, I want to be able to review my input for the export contract again, So that I can do a final review of the information I previously input before submitting my application', () => {
```
Comment (Zainzzkk) [inline e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-minimal.spec.js:16]: Should we add summary list to the context?

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-fully-populated-with-agent-charges-fixed-sum.spec.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  CHECK_YOUR_ANSWERS: { EXPORT_CONTRACT },
+} = INSURANCE_ROUTES;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswers;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Check your answers - Export contract - As an exporter, I want to be able to review my input for the export contract again, So that I can do a final review of the information I previously input before submitting my application - application over total contract value threshold, private insurance attempt, using an agent, agent is chraging - fixed sum method', () => {
```
[the remark this answers is not in the document]
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-fully-populated-with-agent-charges-fixed-sum.spec.js:16]: Good shout, I think we will have this in a lot places and outside of this area (non application check your answers, non Export contract related tests). Will create a ticket for this

[the remark this answers is not in the document]
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-fully-populated-with-agent-charges-fixed-sum.spec.js:16]: Ticket created => https://ukef-dtfs.atlassian.net/browse/EMS-3259

File: e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-minimal.spec.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/export-contract/summary-list/check-your-answers-summary-list-minimal.spec.js:16]: Good shout, added in.

**Review by Zainzzkk — APPROVED** [review]

**Pull request merged**